### PR TITLE
Javascript/javascript2/week3

### PR DESCRIPTION
**Pull request by soujanyapolavarapu**

### CODE REVIEW

File: javascript/javascript2/week3/The-fastest-presser-in-this-realm.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      target: 'my-canvas-two'
+    };
+    var confetti = new ConfettiGenerator(confettiSettings);
+    confetti.render();
+  } else if (countS === 0 && countL === 0) {
+    gameInfo.innerHTML = "No key presses!Try again";
+  } else if (countS === countL) {
+    gameInfo.innerHTML = "Its a Tie ";
+    console.log('Its a Tie');
+
```
Comment (NayanaKamtekar) [inline javascript/javascript2/week3/The-fastest-presser-in-this-realm.js:29]: I think you are confused between `countOne,  countTwo ` and `countS, countL`.  `countOne,  countTwo ` have global scope with press count. `countS, countL` have local scope within the keypress function.  Try to fix game draw and no key press condition.

File: javascript/javascript2/week3/The-fastest-presser-in-this-realm.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+button.addEventListener('click', function gameStart() {
+  let time = userInput.value;
+  //  console.log(Number(time));
+  if (time === "" || Number.isNaN(Number(time))) {
+    gameInfo.innerHTML = "Enter time in seconds"
+  } else {
+    gameInfo.innerHTML = "Game will start and Time limit is set for " + time + " seconds";
+
```
Comment (NayanaKamtekar) [inline javascript/javascript2/week3/The-fastest-presser-in-this-realm.js:40]: Once countdown started no need to show the message "Game will start and time limit is set for x seconds". 
Use `HTMLElement.hidden` property at appropriate place.

File: javascript/javascript2/week3/Warmup.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+runAfterDelay(10,logsOutEarth/*function(){console.log('should logged after 5 seconds')}*/);
+
+//Check if we have double clicked on the page. A double click is defined by two clicks within 0.5 seconds.
+//If a double click has been detected, log out the text: "double click!"
+const functionDoubleClick = ()=>{
+  console.log('double click!');
+    //setTimeout(()=>{console.log('double click!')},0.5 *1000);
+  }
+document.getElementById('button-three').addEventListener('dblclick', functionDoubleClick);
+
```
Comment (NayanaKamtekar) [inline javascript/javascript2/week3/Warmup.js:79]: Its easy to log out the text: "double click!" using `dblclick `event. Try to rewrite using `click `event and `setTimeout.` It definitely takes little more efforts:)

**Review by NayanaKamtekar — APPROVED** [review]
There is scope for improvement in fastest-presser game.
Once game (countdown )starts make input field readonly and remove eventListener from start button so user cannot enter the time and avoid unusual behavior which triggers once start button get clicked while game is going on.

File: javascript/javascript2/week3/Warmup.js (hunk 3 above)
Comment (soujanyapolavarapu) [reply javascript/javascript2/week3/Warmup.js:79]: Thank you for your comments :).Sorry for late response, I will update now.

**Pull request merged**